testproject/Packages/manifest.json

```diff
@@ -6,10 +6,11 @@
     "com.unity.ide.vscode": "1.2.3",
     "com.unity.multiplayer.mlapi": "file:../../com.unity.multiplayer.mlapi",
     "com.unity.multiplayer.transport.utp": "file:../../com.unity.multiplayer.transport.utp",
+    "com.unity.transport": "file:../../../com.unity.transport",
```

Comment:
We can't assume that customers will have this package installed, nor that it would be at this exact location. The transport package is already included in the `com.unity.multiplayer.transport.utp` package, so I would prefer we keep this as the source of truth.

It does mean that while the UTP PR is not approved and the actual package updated on the registry, devs will need to make a change manually in the project though.

Reply:
100% i accidentally committed that change. Thanks for catching it!